themes-default/slim/static/js/vue-submit-form.js

```diff
@@ -0,0 +1,21 @@
+window.vueSubmitForm = async function(formId) {
```

Comment:
I would suggest renaming so we don't get it mixed up with Vue plugins which are also mounted on window and start with "vue".

Reply:
Did I change this? Can't remember doing anything with vue-submit-form.js?

Reply:
@sharkykh did you add it somewhere? Then maybe I rebased it into here?

Reply:
This is from the quality PR
It was there when I rebased